cfme/tests/physical_infrastructure/api/test_physical_server_events_api.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils import testgen
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.wait import wait_for
+
+pytestmark = [pytest.mark.tier(3)]
+
+# The max time in which the full test must complete
+# in seconds (5 min)
+TIMEOUT = 300
+
+# The time interval to perform a verification of state change
+# in seconds (1 min)
+DELAY = 60
+
+# All sources from all currently available physical infrastructure
+# providers
+SOURCES=['LenovoXclarity']
+
+pytest_generate_tests = testgen.generate([LenovoProvider], scope="class")
```

Comment:
**REQUIRED**
Please remove this line and change `pytestmark`:
```python
pytestmark = [
    pytest.mark.tier(3),
    pytest.mark.provider([LenovoProvider], scope="module")
]

Reply:
Thanks @quarckster . I think I have addressed all your comments.